cic_dev/addon/setup-aic

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+# author: sundar.gnanasekaran@intel.com
+
+
+echo "Enter root user password:"
+read a
+
+export password=$a
+export AIC_WORK_DIR=${PWD}
+
+#Install Feature set packages / modules
+./pre-requisites/setup-feature
+
+#Initiate Early-Launch
+./pre-requisites/launch-early
+
+#Remove un-used apks
+./pre-requisites/remove-app
+
+#Install Kernel
+./pre-requisites/setup-kernel
```

Comment:
Why do we install host kernel by default?

Reply:
setup-aic is one time activity. Once done, we just have to use cfc/aic to start or stop container.
Also, We have dependency to use 4.19.80 kernel from chrome always.